Source/Core/Core/DiscordPresence.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DiscordPresence.h"
+#include "discord-rpc/include/discord_rpc.h"
+#include <ctime>
+
+namespace Discord
+{
+void Init()
+{
+  DiscordEventHandlers handlers = {0};
+  Discord_Initialize("450033159212630028", &handlers, 1, NULL);
```

Comment:
Where does this number come from?

Reply:
that number is my Client ID, you need that to initialize Discord's Rich Presence. The Application is used for pictures, and the name of the application.